test/test_jit.py

```diff
@@ -1594,6 +1594,38 @@ def test_if_while(a, b):
         outputs = self._make_scalar_vars([-4321], np.int32)
         self.checkScript(script, inputs, outputs[0], True, 'test_if_while')
 
+    def test_script_for_in_range(self):
+        script = '''
+        def test_for_in_range():
+            c = 0
+            for i in range(100):
+                c += i
+            return c
+        '''
+        self.checkScript(script, [], [4950], True, 'test_for_in_range')
+
+    def test_script_for_in_range_dynamic(self):
+        script = '''
+        def test_script_for_in_range_dynamic():
+            c = 0
+            for i in range(100):
+                acc = 0
+                for j in range(i):
+                    acc += j
+                c += acc
+            return c
+        '''
+        self.checkScript(script, [], [161700], True, 'test_script_for_in_range_dynamic')
+
+    def test_script_bool_constant(self):
+        script = '''
+        def test_script_bool_constant():
+            a = True
+            return a
+        '''
+        outputs = [True]
```

Comment:
Huh? How do we return `True` from a script function? Won't it be a scalar tensor with a 1 inside?

Reply:
Yeah, I'll change it. This is probably not doing what I expected it to do